src/drivers/wdm/queries/StrSafe/StrSafe.ql

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * @name StrSafe
+ * @kind problem
+ * @description Kernel Mode drivers should use ntstrsafe.h, not strsafe.h. The header ntstrsafe.h contains versions of the functions found in strsafe.h that are suitable for use in kernel mode code.

```

Comment:
Can you add a reference to the CA query ID this is porting? i.e. C28146

Reply:
There is a reference in the .qhelp file but I will add it in the description too.